src/views/Home.jsx

```diff
@@ -11,7 +11,12 @@ export function Home({ data, setListPath }) {
 				{/* Renders the `lists` array so we can see which lists the user has access to.  */}
 				{data.map((list) => {
 					return (
-						<SingleList key={list.name} name={list.name} path={list.path} />
+						<SingleList
+							key={list.name}
+							name={list.path}
+							path={list.name / list.path}
```

Comment:
Almost there!

However, the expression `list.name / list.path` reads as "list name divided by list.path", which javascript tries to interpret as a numerical operation as the resultant path that gets set is `NaN`.

You want to concatenate the strings with the `/` in the middle.